components/cards/cards.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<script
+			src="https://kit.fontawesome.com/4e7a84495e.js"
+			crossorigin="anonymous"
+		></script>
+		<link
+			rel="icon"
+			type="image/png"
+			sizes="32x32"
+			href="../../assets/favicon-32x32.png"
+		/>
+		<link rel="stylesheet" href="../../style.css" />
+		<link rel="stylesheet" href="./cards.css" />
+		<title>Cards</title>
+	</head>
+	<body>
+		<div class="container">
+			<nav>
+				<a href="../../index.html" class="logo md-text">Meta UI</a>
+				<section class="nav-right md-text">
+					<a href="https://github.com/rahullrawat" target="_blank"
+						><i class="fab fa-github"></i
+					></a>
+					<a href="https://twitter.com/rahul_rawat08" target="_blank"
+						><i class="fab fa-twitter"></i
+					></a>
+					<a
```

Comment:
Code indentation looks a bit messed up here. You could fix this.

Reply:
I do used prettier for Indentation

Reply:
Yes, thats the issue with prettier that it messes the indentation with `i` tag. You have to manually fix those.

Reply:
ok i will do this manually